src/pyggdrasil/visualize/_mcmc.py

```diff
@@ -138,72 +136,89 @@ def _save_top_trees_plots(data: PureMcmcData, output_dir: Path) -> None:
         plot_tree_mcmc_sample(each, output_dir, save_name=f"top_tree_{rank}")
         rank += 1
 
-
-def make_mcmc_run_panel(
-    data: PureMcmcData,
-    true_tree: TreeNode,
-    similarity_measure: TreeSimilarityMeasure,
-    out_dir: Path,
-) -> None:
```

Comment:
Abandon - rather have single-responsibility functions and generate all plots of interest with a snakemake rule.